extensions/sql-bindings/src/common/constants.ts

```diff
@@ -69,11 +69,10 @@ export const connectionProfile = localize('connectionProfile', 'Select a connect
 export const userConnectionString = localize('userConnectionString', 'Enter connection string');
 export const selectConnectionString = localize('selectConnectionString', 'Select SQL connection string method');
 export const includePassword = localize('includePassword', 'Do you want to include the password from this connection in your local.settings.json file?');
-export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter the password to be used for the connection string');
-export const enterPasswordManually = localize('enterPasswordManually', 'Enter password or press escape to cancel');
 export const userPasswordLater = localize('userPasswordLater', 'In order to user the SQL connection string later you will need to manually enter the password in your local.settings.json file.');
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
+export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter connection password to save in local.settings.json');
```

Comment:
Maybe add an `(Optional)` at the beginning too - I feel like right now this isn't clear enough that they have a choice (and it's shorter than what you had before)